### PR TITLE
feat: allow pr body input via dialog or git editor

### PR DESCRIPTION
**Pull request by vincentvanderweele**
As discussed in #43: add a configuration option to allow the user to specify a custom PR description.

I took the liberty to also add an option to write a multi-line description by launching the git editor, that would suit my use case better than a single-line description.

### CODE REVIEW

**Comment by KnisterPeter** [comment]
Awesome. I'll review the code and if its fine I'll merge and release.

File: package.json [hunk 1]
```diff
@@ -101,6 +112,7 @@
   "dependencies": {
     "execa": "0.5.0",
     "lru-cache": "4.0.2",
+    "mz": "2.6.0",
```
Comment (KnisterPeter) [inline package.json]: What do you need mz for? vscode is running with node6 which supports most es2015/es2016 features and asyc functions are enabled through typescript.

File: src/git.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import * as execa from 'execa';
+import { readFile, unlink } from 'mz/fs';
```
Comment (KnisterPeter) [inline src/git.ts]: For FS you should use 'sander'. This is already a dependency to vscode-github

Comment (vincentvanderweele) [inline src/git.ts]: This is what I needed `mz` for :) Will replace it by sander 👍

Comment (KnisterPeter) [inline src/git.ts]: Great. Thank you.

**Comment by vincentvanderweele** [comment]
Sander's `readFile` seems to return a buffer instead of a string. I suspect `JSON.parse` used to do an implicit `toString`, so everything worked earlier. Since I return the file contents directly I had to change the return type of `readFile` to buffer and transform to string explicitly. Hope that's fine!

**Comment by KnisterPeter** [comment]
Awesome :rocket: 
I'll do a release later today.

**Pull request merged**